tests/bin/sev_get_att_quote.c

```diff
@@ -0,0 +1,34 @@
+#include "libc.h"
+#include <errno.h>
+
+/* This test will be run only for KVM. It is designed to request a
+ * Quote from get_attestation() and check that the first bytes of
+ * the returned Quote in buf match expected values. */
+
+int main(void) {
```

Comment:
This test will soon be "baked in" to the unattested launch feature I'm working on. We can't easily do the remote attestation protocol within our barebones libc here. I think this file can be removed.

Reply:
ok